crates/cli/src/codegen/mod.rs

```diff
@@ -50,18 +50,23 @@ use wasm_opt::{OptimizationOptions, ShrinkLevel};
 use wasmtime_wasi::{pipe::MemoryInputPipe, WasiCtxBuilder};
 use wizer::{Linker, Wizer};
 
-use crate::{js_config::JsConfig, plugins::Plugin, JS};
 use anyhow::Result;
 
 static STDIN_PIPE: OnceLock<MemoryInputPipe> = OnceLock::new();
 
-pub(crate) enum CodeGenType {
-    /// Static code generation.
```

Comment:
It's admittedly a little silly but I'd like we have triple slash comments on anything that'll be public. There are clippy settings and other lint things that flag missing triple-slash comments on public parts of the API. And this will at some point be public again. Granted the comment will need to be updated to reflect the new enum name.